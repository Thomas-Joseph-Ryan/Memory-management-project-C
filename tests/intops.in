min a
max a
sum a
len a
set a 1 2 3 4 5 6
min a
max a
sum a
len a
set b a 5 6 7 8
min b
max b
sum b
len b