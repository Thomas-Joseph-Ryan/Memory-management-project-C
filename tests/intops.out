> no such key

> no such key

> no such key

> no such key

> ok

> 1

> 6

> 21

> 6

> ok

> 1

> 8

> 47

> 10

> 
bye
